client/payment-details/dispute-details/dispute-footer.tsx

```diff
@@ -0,0 +1,187 @@
+						dispute.metadata.__evidence_submitted_at,
+						'M j, Y'
+				  )
+				: '-';
+
+			buttonLabel = __(
+				'View submitted evidence',
+				'woocommerce-payments'
+			);
+			message = createInterpolateElement(
+				sprintf(
+					isInquiry( dispute )
+						? /* Translators: %s - formatted date, <a> - link to documentation page */
+						  __(
+								'You submitted evidence for this inquiry on %s. The cardholder’s bank is reviewing the case, which can take 120 days or more. You will be alerted when they make their final decision. <a>Learn more</a>.',
+								'woocommerce-payments'
+						  )
+						: /* Translators: %s - formatted date, <a> - link to documentation page */
+						  __(
+								'You submitted evidence for this dispute on %s. The cardholder’s bank is reviewing the case, which can take 60 days or more. You will be alerted when they make their final decision. <a>Learn more about the dispute process</a>.',
+								'woocommerce-payments'
+						  ),
+					submissionDateFormatted
+				),
+				{ a: disputeDocsLinkElement }
+			);
+			break;
+		case 'lost':
+			if ( isAccepted ) {
+				// Lost accepted
+				message = createInterpolateElement(
+					sprintf(
+						/* Translators: %1$s - formatted date, %2$s – the formatted dispute fee amount, <a> - link to documentation page */
+						__(
+							'This dispute was accepted and lost on %1$s. The %2$s fee has been deducted from your account, and the disputed amount returned to the cardholder. <a>Learn more about preventing disputes</a>.',
+							'woocommerce-payments'
+						),
+						closedDateFormatted,
+						disputeFeeFormatted
+					),
+					{ a: disputeDocsLinkElement }
+				);
+			} else if ( isSubmitted ) {
+				// Lost
+				message = createInterpolateElement(
+					sprintf(
+						/* Translators: %1$s - formatted date, %2$s – the formatted dispute fee amount, <a> - link to documentation page */
+						__(
+							'This dispute was lost on %1$s. The %2$s fee has been deducted from your account, and the disputed amount returned to the cardholder. <a>Learn more about preventing disputes</a>.',
+							'woocommerce-payments'
+						),
+						closedDateFormatted,
+						disputeFeeFormatted
+					),
+					{ a: disputeDocsLinkElement }
+				);
+			} else {
+				// Lost no response
+				message = createInterpolateElement(
+					sprintf(
+						/* Translators: %1$s - formatted date, %2$s – the formatted dispute fee amount, <a> - link to documentation page */
+						__(
+							'This dispute was lost on %1$s due to non-response. The %2$s fee has been deducted from your account, and the disputed amount returned to the cardholder. <a>Learn more about preventing disputes</a>.',
+							'woocommerce-payments'
+						),
+						closedDateFormatted,
+						disputeFeeFormatted
+					),
+					{ a: disputeDocsLinkElement }
+				);
+			}
+			break;
+	}
+
+	const handleClick = () => {
+		wcpayTracks.recordEvent(
+			wcpayTracks.events.TRANSACTION_DETAILS_DISPUTE_FOOTER_BUTTON_CLICK,
```

Comment:
This event feels generic – what questions could we answer with it? Would it be better to have unique events for each type of action button, so the user action is captured in the name? 

If I asked the user what they were doing, I think they would say "I wanted to review the evidence to see why I lost the dispute" rather than "I clicked the transaction details footer button". 

(Related to component naming)

Reply:
Also related to suggestion to use sub-components for each 'case'.